submissions/andrewklmn/html-css-popup/app.css

```diff
@@ -51,54 +17,118 @@ nav {
   justify-content: flex-end;
 }
 
-nav a {
+nav > ul > li > a {
```

Comment:
Avoid creating too specific selectors. In a big project, it's hard to support or override by another selector.
Use `class` for adding styles to element.
[Read more about selector specificity.](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)